tests/app/questionnaire/test_questionnaire_schema.py

```diff
@@ -1090,3 +1090,27 @@ def test_grand_calculated_summary_when_rule_dependencies():
             "q2-a2": {"section-4"},
         }
     )
+
+
+def test_placeholder_transform_section_dependencies_by_block_for_calculation_summaries():
+    """
+    Ensures that dependencies are captured correctly for calculation summary blocks using transforms.
+    In this schema the calculation summaries use placeholder transforms based on other blocks that have dependencies in the
+    reporting-period-section
+    """
+    schema = load_schema_from_name(
+        "test_placeholder_dependencies_with_calculation_summaries"
+    )
+
+    assert (
+        schema._placeholder_transform_section_dependencies_by_block  # pylint: disable=protected-access
```

Comment:
This should be using the public property instead of the private attribute.

Reply:
Updated 👍